### DISCUSSION TITLE
Resync winget database after an app is updated externally.

### DISCUSSION THREAD

**Discussion by WhoDunDidIt** [microsoft/winget-cli#3634] [Q&A]
For example, winget shows me this:

Name       Id                   Version Available Source
\--------------------------------------------------------
OBS Studio OBSProject.OBSStudio 29.0.2  29.1.3    winget

However, the currently installed version of OBS **__is__** 29.1.3.

How do I get winget to rescan installed apps and their versions?

**Comment by Masamune3210** [comment]
You misunderstand. Winget doesnt keep its own database of installed software and their version, it uses the system's database, the same one that is used for the Add/Remove Programs/Settings dialogs. If the version is being misreported by winget, its because the app failed to properly update the value in the registry like its supposed to.
